4a-seurat_normalize.R

```diff
@@ -0,0 +1,97 @@
+# CCDL for ALSF 2018
+# C. Savonen
+#
+# Purpose: Using Seurat for post-processing of scRNA-seq Salmon processed data
```

Comment:
Specific to Salmon or could you use this in other contexts with any count data?

Reply:
Am I correct in that this uses count data?

From [the tutorial](https://satijalab.org/seurat/get_started.html): 
> The input to Seurat is a gene expression matrix, where the rows are genes and the columns are single cells.

Reply:
Yeah, it can be used with other data as long as its a gene matrix. Will change the documentation to reflect that.